zap/zap.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package zap;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = "github.com/bigbluedisco/protoc-gen-zap/zap";
+
+enum ObfuscationType {
+    OBFUSCATION_TYPE_UNSPECIFIED = 0;
+    HIDE = 1; // Remove field and value from log entierly
+    STARS = 2; // Replace value by 3 stars
+}
+
+extend google.protobuf.FieldOptions {
+    ObfuscationType obfuscation_type = 50002;
```

Comment:
Can I suggest a shorter name? I dont think we need _type, obfuscate or obfuscation work

```suggestion
    ObfuscationType obfuscate = 50002;
```